src/rules/commentFormatRule.ts

```diff
@@ -173,19 +239,37 @@ function walk(ctx: Lint.WalkContext<Options>) {
         if (start === end) {
             return;
         }
-        const commentText = fullText.slice(start, end);
+        let commentText = fullText.slice(start, end);
         // whitelist //#region and //#endregion and JetBrains IDEs' "//noinspection ..."
         if (/^(?:#(?:end)?region|noinspection\s)/.test(commentText)) {
             return;
         }
 
+        if (ctx.options.noSpaceExceptions !== undefined) {
+            const match = ctx.options.noSpaceExceptions.exec(commentText);
+            if (match !== null) {
+                const lengthOfMatch = match[0].length;
+                commentText = commentText.substring(lengthOfMatch);
```

Comment:
Nit suggestion: you only use the `lengthOfMatch` variable once, so just say `commentText.substring(match[0].length)` to have fewer lines.

Reply:
Oh, I'm silly. Thanks!

Reply:
😄